staging/src/k8s.io/apiserver/pkg/apis/apiserver/types.go

```diff
@@ -215,7 +215,7 @@ const (
 	TypeWebhook                                      AuthorizerType = "Webhook"
 	FailurePolicyNoOpinion                           string         = "NoOpinion"
 	FailurePolicyDeny                                string         = "Deny"
-	AuthorizationWebhookConnectionInfoTypeKubeConfig string         = "KubeConfigFile"
```

Comment:
KubeConfigFile matches the name of the field where the filename is specified, and matches the existing admission config field name. I would keep this as-is and update the KEP

Reply:
In that case, I would update the constant name.

I am okay with either of the ways.

Reply:
sounds good, let's update the constant

Reply:
this is still outstanding

Reply:
this should be done now